onnxruntime/contrib_ops/cuda/bert/attention_impl.h

```diff
@@ -46,8 +46,8 @@ Status LaunchAttentionKernel(
     void* workspace,                           // Temporary buffer
     void* output,                              // Output tensor
     void* present,                             // Present state output
-    void* fused_runner                         // Fused multi-head attention
-);
+    void* fused_runner,                        // Fused multi-head attention
+    const std::vector<int64_t> qkv_sizes = {});  // Hidden sizes of qkv
```

Comment:
This input is not needed. qkv_sizes can be deduced from qkv_head_size * num_heads.